pkg/controller/configmap/configmap_controller.go

```diff
@@ -0,0 +1,133 @@
+	if err != nil {
+		return err
+	}
+
+	// Watch for changes to primary resource Group
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{}, predicate.Funcs{
+		CreateFunc: func(createEvent event.CreateEvent) bool {
+			return createEvent.Meta.GetName() == userCABundleConfigMapName && createEvent.Meta.GetNamespace() == names.ADDL_TRUST_BUNDLE_CONFIGMAP_NS
+
+		},
+		DeleteFunc: func(deleteEvent event.DeleteEvent) bool {
+			return deleteEvent.Meta.GetName() == userCABundleConfigMapName && deleteEvent.Meta.GetNamespace() == names.ADDL_TRUST_BUNDLE_CONFIGMAP_NS
+		},
+		UpdateFunc: func(updateEvent event.UpdateEvent) bool {
+			return updateEvent.MetaNew.GetName() == userCABundleConfigMapName && updateEvent.MetaNew.GetNamespace() == names.ADDL_TRUST_BUNDLE_CONFIGMAP_NS
+		},
+		GenericFunc: func(genericEvent event.GenericEvent) bool {
+			return genericEvent.Meta.GetName() == userCABundleConfigMapName && genericEvent.Meta.GetNamespace() == names.ADDL_TRUST_BUNDLE_CONFIGMAP_NS
+		},
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// blank assignment to verify that ReconcileGroup implements reconcile.Reconciler
+var _ reconcile.Reconciler = &ReconcileConfigMap{}
+
+// ReconcileConfigMap reconciles a ConfigMap object
+type ReconcileConfigMap struct {
+	client            client.Client
+	scheme            *runtime.Scheme
+	metricsAggregator *metrics.AdoptionMetricsAggregator
+}
+
+// Reconcile reads that state of the cluster for a ConfigMap object and makes changes based the contained data
+func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	reqLogger.Info("Reconciling ConfigMap")
+
+	// Fetch the ConfigMap openshift-config/user-ca-bundle
+	cfgMap := &corev1.ConfigMap{}
+	ns := names.ADDL_TRUST_BUNDLE_CONFIGMAP_NS
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: ns, Name: userCABundleConfigMapName}, cfgMap)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+	}
+
+	certBundle, _, err := validation.TrustBundleConfigMap(cfgMap)
+	if err != nil {
+		// Reporting of failed validation is already funneled through CVO via ClusterOperatorDegraded. This is to explicity catch this condition
+		// and handle gracefully rather then causing stacktrace.
+		if strings.Contains(err.Error(), "failed to parse certificate PEM") {
+			reqLogger.Info("failed to parse certificate PEM")
+			r.metricsAggregator.SetClusterProxyCAValid(false)
+			reqLogger.Info("metric should be set here meow")
```

Comment:
```suggestion
			reqLogger.Info("setting CA valid metric to false")
```

Reply:
:facepalm: this was from debugging. Will commit your suggestion.